src/main/java/org/mule/service/http/impl/service/server/HttpServerManager.java

```diff
@@ -63,12 +85,36 @@ HttpServer createServerFor(ServerAddress serverAddress, Supplier<Scheduler> sche
    * @return the create Server handler
    * @throws ServerCreationException if it was not possible to create the Server. Most likely because the host and port is already
    *                                 in use.
+   * @deprecated as of 1.6.0 use
```

Comment:
These 1.6.0 all refer to the http-connector version. Please change them to the corresponding http-service version.
Also, add the `@since` to the new api